src/root/components/user-card/user-card.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { Serializable } from '@features/authentication';
+
+type User = {
+  username: Serializable;
+  groups: Serializable;
+};
+
+@Component({
+  selector: 'app-user-card',
+  templateUrl: './user-card.component.html'
+})
+export class UserCardComponent {
+  @Input() public user?: User | null;
```

Comment:
```suggestion
  @Input({ required: true }) public user?: User | null;
```
Depuis Angular 16 ;)